templates/new_home/index.html

```diff
@@ -7,18 +7,36 @@
 {% block style %}
 <!--Favicon-->
 <link rel="icon" type="text/css" href="{% static 'new_home/images/logo.png' %}">
-
-<link rel="stylesheet" href="{% static 'new_home/style.css' %}" type="text/css" />
-<script src="{% static 'new_home/style.js' %}" type="text/javascript" defer></script>
-
 <!-- Bootstrap & Downloaded CSS/JS -->
 <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
     integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
 <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+     integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w==" crossorigin="anonymous" />
 <script src="https://unpkg.com/aos@2.3.1/dist/aos.js"></script>
+<!-- castome CSS -->
+<link rel="stylesheet" href="{% static 'new_home/style.css' %}" type="text/css" />
+<script src="{% static 'new_home/style.js' %}" type="text/javascript" defer></script>
+
 {% endblock %}
 
 {% block content %}
+
+<!-- alert section starts -->
```

Comment:
I think alerts should be below the navigation bar.  What do you say @garg3133?

Reply:
Done changes !